plugins/default/lib/disk_usage.py

```diff
@@ -1,35 +1,97 @@
 #!/usr/bin/env python
 import re
 import sys
+import json
+import argparse
 from subprocess import CalledProcessError
+from collections import Counter
 
-from openshift import oc
+import openshift
 import nagios
 
 
 check_disk_cmd = ("df", "--output=target,pcent,ipcent")
 # Example output:
 # /etc/hosts      19%    8%
-check_disk_output_pattern = re.compile(r"([\w/]+)\s+(\d+)%\s+(\d+)%")
+check_disk_output_pattern = re.compile(
+    r"([A-Za-z0-9\./\-_]+)\s+(\d+)%\s+(\d+)%")
```

Comment:
Because it missed several file systems in my testing.  Examples are:
/etc/feedhenry/gitlab-shell (because of the -)
/run/secrets/kubernetes.io/serviceaccount (because of the .)